checkmk_extensions/agent/scheduler_runner.ps1

```diff
@@ -27,3 +27,25 @@ class Config {
         return [Config]::new($use_rcc)
     }
 }
+function CreateCommand([Config]$config) {
+    $parentDir = Split-Path $PSScriptRoot -Parent
+
+    if ($config.rcc) {
+        $rccPath = Join-Path $PSScriptRoot "rcc.exe"
+        $robotPath = Join-Path $parentDir "config\robot.yaml"
+
+        $argumentsList = "run --robot $robotPath"
+    }
+    else {
+        $pythonExe = (Get-Command python).Source
+        $moduleName = "robotmk.scheduler"
+        $configPath = Join-Path $parentDir "config\robotmk.json"
+
+        $argumentsList = "-m $moduleName --config $configPath"
+    }
+
+    return [PSCustomObject]@{
+        Executable = if ($config.rcc) { $rccPath } else { $pythonExe }
+        Arguments = $argumentsList
```

Comment:
Is this now an actual list (sorry, my PowerShell syntax knowledge is very poor ..)? Basically, it has to work as input for
[-ArgumentList] <String[]> in Start-Process, right?

Reply:
I have a commit ready that will make a list out of this, because at this point it's just a long string. But, Start-Process also works with a string as well.

Reply:
Ok, then it can also stay as is